_store/liaraalis-small-business-rule-bundle.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Small business regulation plugin for Kimai 2
+name: Small business regulation plugin
+intro: "Kimai 2 plugin that provides the possibility to use the small business regulation, as it can be applied in Germany and Austria, when creating invoices in Kimai 2."
```

Comment:
```suggestion
intro: "A plugin that provides the possibility to use the small business regulation, as it can be applied in Germany and Austria, when creating invoices in Kimai."
```